content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -53,25 +66,64 @@ Make sure that you have [Go](https://go.dev/) installed and that your `GOPATH` i
     ...
     ```
 
-    The go-car package makes it easier to work with content-addressed archive (CAR) files.
+   The go-car package makes it easier to work with content-addressed archive (CAR) files.
 
-You now have everything you need to retrieve a file with Lassie and extract the contents with Go-car.
+You now have everything you need to retrieve a file with Lassie and extract the contents with `go-car`.
 
-### Retrieve
+#### Retrieve
 
-To retrieve data from Filecoin using Lassie, all you need is the CID of the content you want to download. You can use the following CIDs to test the process:
+To retrieve data from Filecoin using Lassie, all you need is the CID of the content you want to download.
 
-1. The format for retrieving data using Lassie is:
+Below video demonstrate how Lassie can be used to render content directly from Filecoin/IPFS.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/h_zCd7ssKCQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-    ```shell
-    lassie fetch -o <OUTFILE_FILE_NAME> -p <CID>
-    ```
+Lassie and go-car can work together to retrieve and extract data from Filecoin, all you need is the CID of the content to download.
 
-    For example:
+```shell
+lassie fetch -o - <CID> | car extract
+```
 
-    ```shell
-    lassie fetch -o output.car -p bafykbzaceatihez66rzmzuvfx5nqqik73hlphem3dvagmixmay3arvqd66ng6
-    ```
+This command uses a `|` to chain two commands together, this will work on Linux or a macOS. Windows users may need to use PowerShell to use this form. Alternatively, you can use the commands separately as explained later in this page.
```

Comment:
```suggestion
This command uses a `|` to chain two commands together. This will work on Linux or macOS. Windows users may need to use PowerShell to use this form. Alternatively, you can use the commands separately as explained later in this page.
```